doc/development.md

```diff
@@ -1,23 +1,5 @@
 # Development Guide
 
-<!--ts-->
```

Comment:
why remove the TOC? :cry:

Reply:
They are generated automatically already in the docs website and by Github.

<img width="1461" alt="Screenshot 2023-11-10 at 15 13 00" src="https://github.com/Kuadrant/limitador-operator/assets/1842261/dd584cbf-3fbc-402f-bbdc-93cdf910a4ef">

<img width="1474" alt="Screenshot 2023-11-10 at 15 15 01" src="https://github.com/Kuadrant/limitador-operator/assets/1842261/2a22270c-7008-4d37-acc1-0ee5d92abfd6">